<launch>

    <!-- Sensors -->

        <!-- LiDAR -->
        <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />

    <!-- Sensors end -->
        
    <!-- Odometry -->
    <node name="fake_odometry" pkg="odometry" type="fakeOdometry.py" output="screen" />

    <!-- Path Planner -->
    <node name="odometry_path_generator" pkg="path_planner" type="odometry_path_generator.py" output="screen" />

    <!-- Localization -->
    <include file="$(find hdl_localization)/launch/hdl_localization.launch">
        <arg name="map_file" value="$(find hdl_localization)/data/Globalmap2.pcd" /> 
    </include>

    <!-- Model -->
    <include file="$(find erp42_vehicle_rviz)/launch/erp42_vehicle_rviz.launch" />

    <!-- Test -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0.0 0.0 0.0 0.0 0.0 0 map odom 10" /> -->

</launch>